.github/wiki/Architecture-Documentation.md

```diff
@@ -0,0 +1,63 @@
+This document outlines the software and hardware requirements for running the infrastructure of the [example app](https://github.com/SanG-github/nimble-devops-ic-web). You can refer to the [[Architecture Diagram]] for a visual overview of the infrastructure components.
+
+## Required Hardware
+
+The infrastructure is hosted on AWS and relies on various AWS services, including ECS with Fargate launch type, RDS, and ElastiCache. Migrating to a different cloud provider is possible, thanks to Terraform's flexibility.
```

Comment:
this infrastructure project target AWS, so I believe if the following statement is correct
> Migrating to a different cloud provider is possible, thanks to Terraform's flexibility.

Reply:
I haven't tried it yet, but those are my thoughts about Terraform 😗 Thus let me remove them to reduce confusion.